docs/_data/previous_versions.yml

```diff
@@ -0,0 +1,6 @@
+- title: 0.14.0
```

Comment:
It's `0.0.14` not `0.14.0`. Also, how are these strings updated when we release new versions?

Reply:
They're not - the script that freezes the version just tells you to do it manually. It's maybe not the nicest but I also didn't want to over-engineer is and though it would be best to put out there and see which bits of the process are pain points...